src/urllib3/__init__.py

```diff
@@ -22,7 +23,6 @@
 
 __author__ = "Andrey Petrov (andrey.petrov@shazow.net)"
 __license__ = "MIT"
-__version__ = "1.25.8"
```

Comment:
We have a lint failure here. Could it be fixed  using `__version__ = __version__`? It would have the benefit of being explicit too.

Reply:
Will change to that!